build/gulp.modules.js

```diff
@@ -93,13 +93,35 @@ const buildModuleBuilder = cb => {
 }
 
 const buildModules = () => {
-  const modules = getAllModulesRoot()
+  const allModules = getAllModulesRoot()
+
+  const command = process.argv[process.argv.length - 2]
+  const moduleArgs = process.argv[process.argv.length - 1].split(',')
+
+  let moduleFilter = m => true
+  if (command === '--m') {
+    moduleFilter = m => moduleArgs.includes(m)
+  } else if (command === '--a') {
+    // if command="--a nlu" we match nlu, nlu-testing and nlu-extras
+    moduleFilter = m => moduleArgs.some(arg => !!m.match(new RegExp(`${arg}`)))
+  }
+
+  const modules = allModules
+    .map(modulePath => {
+      const config = readModuleConfig(modulePath)
+      const moduleName = _.get(config, 'name', 'Unknown')
+      return {
+        modulePath,
+        moduleName
+      }
+    })
+    .filter(({ moduleName }) => moduleFilter(moduleName))
```

Comment:
This block is not necessary, the name of the module is always the same as its folder, so you just need to get the last part of the paths. ex: `allModules.map(x => path.basename(x)`.

Also, can you add an entry on the cheat sheet on `gulpfile.js` ?

Reply:
good! I'm gonna make the changes + update the cheat sheet!